### ISSUE TITLE
Add Source Download Option (USFM & OBS)

### ISSUE THREAD

**Issue by jag3773** [unfoldingWord-dev/door43.org#71]
## Story

As a _translator_ I want to download my translation as a Resource Container USFM zip bundle so that I can use it in other tools.

As a _translator_ I want to download my translation as a Resource Container bundle (for OBS) so that I can use it in other tools.

http://resource-container.readthedocs.io/en/latest/structure.html

## Notes

Originally https://github.com/unfoldingWord-dev/gogs/issues/14.  Assuming we have a single "Download" button on the project page ( #58 ) then we could add this to the drop down list.

Effectively this will need to concatenate the chunks together to form a valid USFM structure (1 file per book) and then put these in a zip file on the CDN for that project (like in #13 ).  See https://github.com/unfoldingWord/ulb-en for an example of a valid USFM structure, or see the USFM reference manual.

**Note** The necessary zip files are already being created, we just need to link to them and/or move them to permanent CDN location and then link to them.

Probably have JS build the link for the button

**Comment by PhotoNomad0** [comment]
Note: From https://github.com/unfoldingWord-dev/gogs/issues/14 it looks like this is an option to add to the live.door43.org page.

**Comment by PhotoNomad0** [comment]
@jag3773 - looking at the preconvert files for our projects.  
- For bible - we have a zip of manifest.json and a single USFM for book
- For OBS - we have a zip of manifest.json and an md file for each chapter

Are these sufficient as download files?

**Comment by jag3773** [comment]
Can you post some examples? Do they conform to the RC spec by chance?

**Comment by PhotoNomad0** [comment]
@jag3773 - short answer is no.  I'll look into what it will take to make it conform.

**Comment by PhotoNomad0** [comment]
@jag3773 
- for OBS: [2a7355b4fb.zip](https://github.com/unfoldingWord-dev/door43.org/files/1022369/2a7355b4fb.zip)
- for EPH: [200ba03344.zip](https://github.com/unfoldingWord-dev/door43.org/files/1022372/200ba03344.zip)

**Comment by PhotoNomad0** [comment]
@jag3773 - the preconvert downloads the repo as a zip with all the structure in place. 

- for bible it merges everything to a single usfm for the book and saves it in zip file with manifest. This looks like it meets the goal above for single USFM but it is not a valid resource container.

For OBS all the chunks are merged into chapters, then it creates a zip file with all the chapters as md files and a manifest file.  This is not a valid resource container.  Perhaps in this case it would be better to just download the repo zip?

**Comment by PhotoNomad0** [comment]
@jag3773 - do we need to build a manifest.yml file?

**Comment by jag3773** [comment]
It *must* include the  manifest.yaml file.  Here is the example for USFM: https://git.door43.org/Door43-Catalog/en_ulb

**Comment by jag3773** [comment]
The links that you posted look good.

**Comment by PhotoNomad0** [comment]
Waiting on https://github.com/unfoldingWord-dev/door43.org/issues/428 in order to add manifest.yaml.

**Comment by jag3773** [comment]
Bruce, I should clarify that it needs to include whatever manifest file is available in the repo. This task should not be blocked by requiring the latest version.

**Comment by jag3773** [comment]
@PhotoNomad0 : @Thajny will upload a screenshot that has the download button and print button on it so you can see how it needs to be styled.

**Comment by jag3773** [comment]
![obs 1](https://cloud.githubusercontent.com/assets/107731/26684191/88c0dd9c-46a3-11e7-8a48-f7c7cbee65a0.png)

@PhotoNomad0 Let's use a zip file icon for the source download entry.  The entry should be called "USFM" for Bible or "Markdown" if it's an OBS project.

**Comment by PhotoNomad0** [comment]
@richmahn - any examples on how to do a drop down menu for Download?

**Comment by richmahn** [comment]
@PhotoNomad0 https://www.w3schools.com/bootstrap/bootstrap_dropdowns.asp

**Comment by PhotoNomad0** [comment]
Styled button similar to @richmahn in https://github.com/unfoldingWord-dev/door43.org/pull/563
